src/Mod/PartDesign/App/FeatureHelix.cpp

```diff
@@ -77,21 +77,22 @@ Helix::Helix()
 {
     addSubType = FeatureAddSub::Additive;
 
-    ADD_PROPERTY_TYPE(Base, (Base::Vector3d(0.0, 0.0, 0.0)), "Helix", App::Prop_ReadOnly, "Base");
-    ADD_PROPERTY_TYPE(Axis, (Base::Vector3d(0.0, 1.0, 0.0)), "Helix", App::Prop_ReadOnly, "Axis");
-    ADD_PROPERTY_TYPE(Pitch, (10.), "Helix", App::Prop_None, "Pitch");
-    ADD_PROPERTY_TYPE(Height, (30.0), "Helix", App::Prop_None, "Height");
-    ADD_PROPERTY_TYPE(Turns, (3.0), "Helix", App::Prop_None, "Turns");
+    const char* group = "Helix";
+    ADD_PROPERTY_TYPE(Base, (Base::Vector3d(0.0, 0.0, 0.0)), group, App::Prop_ReadOnly, "The center point of the helix' start.");
+    ADD_PROPERTY_TYPE(Axis, (Base::Vector3d(0.0, 1.0, 0.0)), group, App::Prop_ReadOnly, "The helix' direction.");
+    ADD_PROPERTY_TYPE(Pitch, (10.), group, App::Prop_None, "The axial distance between two turns.");
+    ADD_PROPERTY_TYPE(Height, (30.0), group, App::Prop_None, "The total length of the helix (not accounting for the extent of the profile)");
+    ADD_PROPERTY_TYPE(Turns, (3.0), group, App::Prop_None, "The number of turns in the helix (does not need to be a whole number).");
     Turns.setConstraints(&floatTurns);
-    ADD_PROPERTY_TYPE(LeftHanded, (long(0)), "Helix", App::Prop_None, "LeftHanded");
-    ADD_PROPERTY_TYPE(Reversed, (long(0)), "Helix", App::Prop_None, "Reversed");
-    ADD_PROPERTY_TYPE(Angle, (0.0), "Helix", App::Prop_None, "Angle");
-    ADD_PROPERTY_TYPE(Growth, (0.0), "Helix", App::Prop_None, "Growth");
+    ADD_PROPERTY_TYPE(LeftHanded, (long(0)), group, App::Prop_None, "Determins whether the turning direction of helix follows the \"left-hand rule\", i.e. counter-clockwise when moving along its axis.");
```

Comment:
Determin_**e**_s

However I don't find this info helpful since one need to know that the rule is. What about just

"Sets turning direction to left-handed"

Those who deal with threads and helices know the rule so we can keep the info short.

Reply:
In my view, the tool tips are especially for those who *don't* know; users who do, don't need a tool tip at all. So in that sense, a tool tip for "Foo" which just tells "sets the Foo" is barely helpful.

What I tried to do with this text here is
a) to let the property name make sense: It's named `LeftHanded` *in reference to* the left-hand rule.
b) to explain what is does: It make the helix turn counter-clockwise when moving along its axis.

(`IsLeftHanded` would be a better property name, but it's too late to change this)

The typo gets fixed, of course. Thanks for pointing out.

Reply:
You explain too left-hand with left-hand.

"counter-clockwise when moving along its axis."

is only helpful for those who knows what left-had means. To make it clear one must mention the axis direction and from what direction one looks at the system. It is in my opinion impossible to explain left-handed in a tooltip and as i wrote, those who deal with helices know it. If not the "turning direction" makes clear what it is about and since there re only 2 possibilities, users can just check them out.